api/README.md

```diff
@@ -6,17 +6,19 @@ This page provides the following information required for you to get started wit
 * [Contact Info](#contact-info)
 * [Environment](#environment)
 * [Authentication](#authentication)
-* [Email Address Normalization](#email-address-normalization)
 * [Query Parameter Value Encoding](#query-parameter-value-encoding)
-* [Email Address Hash Encoding](#email-address-hash-encoding)
+* [Email Address Normalization](#email-address-normalization)
+* [Address Hash Encoding](#email-address-hash-encoding)
```

Comment:
`Email Address Hash Encoding`? Looks like the word `Email` has been lost here.

Reply:
fixed.